tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -5823,10 +5851,10 @@ def get_prod(arr):
     integer32 = randint(min_int32, max_int32, size = size, dtype=np.int32)
     integer64 = randint(min_int64, max_int64, size = size, dtype=np.int64)
 
-    fl = uniform(min_float / 2, max_float / 2, size = size)
-    fl32 = uniform(min_float32, max_float32, size = size)
+    fl = uniform(min_int, max_int, size = size)
+    fl32 = uniform(min_int, max_int, size = size)
     fl32 = np.float32(fl32)
-    fl64 = uniform(min_float64 / 2, max_float64 / 2, size = size)
+    fl64 = uniform(min_int64, max_int64, size=size)
 
     cmplx128_from_float32 = uniform(low=-((-min_float32) ** (1/5)), high=(max_float32 ** (1/5)), size = size) + uniform(low=-((-min_float32) ** (1/5)), high=(max_float32 ** (1/5)), size = size) * 1j
```

Comment:
Why do you only have the calculation of the largest possible value for complexes?

Reply:
it seems dump, but the results of floats and ints never pass the bounds.

Reply:
Which bounds is that true for? The old bounds (e.g for f164: `min_float64 / 2`) or the new bounds ( e.g. for f164 `min_int64`)?
For the new bounds that seems likely as floats can contain much larger numbers than integers. You are therefore a long way from testing the full range of floats at the moment (this may not be a problem, you are still testing a large range of floats)